installing_on_ibm_cloud/_topics/installation.md

```diff
@@ -0,0 +1,175 @@
+## Installing {{ site.data.product.title }}
+
+Installing {{ site.data.product.title }} consists of the following steps:
+
+1.  Downloading the appliance for your environment as a virtual machine image template.
+
+2.  Setting up a virtual machine based on the appliance.
+
+3.  Configuring the {{ site.data.product.title_short }} appliance.
+
+After you have completed all the procedures in this guide, you will have a working environment on which additional customizations and configurations can be performed.
+
+### Obtaining the {{ site.data.product.title }} Virtual Appliance
+
+### Uploading and Provisioning the {{ site.data.product.title }} Virtual Appliance in IBM Cloud
+
+To upload the {{ site.data.product.title }} appliance file in IBM Cloud, ensure the following requirements are met:
+
+  - Approximately 2 GB of space for each image; 44+ GB of space, 12 GB RAM, and 4 VCPUs for the {{ site.data.product.title }} appliance.
+
+  - You must have an IBM Cloud user account with the following roles: 
+![Figure showing the required roles for an IBM Cloud user account.](../images/required_roles.png){: caption="Figure 1. Required roles for IBM Cloud user account" caption-side="bottom"}
+
+  - Depending on your infrastructure, allow time for the upload.
+
+### Step A. Setting up the Custom image for {{ site.data.product.title }} in IBM Cloud
+{: #config-image}
+
+Create a custom Linux-based image to deploy {{ site.data.product.title }} as a virtual server instance in IBM Cloud.
+
+1. If you don't already have an instance of IBM Cloud Object Storage, see [Getting started with IBM Cloud Object Storage](https://cloud.ibm.com/docs/services/cloud-object-storage?topic=cloud-object-storage-getting-started)
+
+    ![Figure showing example Cloud Object Storage created.](../images/cloud_object_storage.png){: caption="Figure 2. Example Cloud Object Storage created" caption-side="bottom"}
+
+    You must also create a bucket in IBM Cloud Object Storage to store your images.
+    ![Figure showing example standard type bucket created.](../images/buckets.png){: caption="Figure 3. Example Standard type bucket created" caption-side="bottom"}
+
+
+2. Upload the {{ site.data.product.title }} installation image (file name: `cfme-rhos-5.11.4.x86_64.qcow2`) to your IBM Cloud Object Storage. Select your bucket and click Add Objects to upload the images. For more information, see [Uploading data by using the console](https://cloud.ibm.com/docs/services/cloud-object-storage?topic=cloud-object-storage-upload#upload-console). **Note:** You can use the Aspera high-speed transfer plug-in to upload images larger than 200 MB.  
```

Comment:
Should the image name be whatever the upstream name is?

Reply:
Good catch. I will update the name AND the image displayed so they both reflect the .q2 file.  stand by.